src/assets/less/components/jumbotron.less

```
@@ -51,31 +55,32 @@
-		@media (min-width: @screen-sm-min) {
-			height: 584px;
+		.jumbotron_blur {
+			display: none;
 		}
 
-		.jumbotron_blur {
-			top: 160px;
-			height: 296px;
+		.jumbotron_globe {
+			--globe-size: min(900px, calc(100vw - 64px));
+			right: 0;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)